cmd/centaurid/cmd/genaccounts.go

```diff
@@ -201,7 +201,7 @@ contain valid denominations. Accounts may optionally be supplied with vesting pa
 func CovertPrefixAddr() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "convert [address]",
-		Short: "Convert prefix from layer to centauri",
+		Short: "Convert prefix from centauri to pica",
```

Comment:
i am not so deep into. should it be layer -> centauri -> pica? I mean 2 converters retained? but I guess you read context, so ok for me.

Reply:
so yes. the changes is for public net to change from centauri to pica.
not for devnet where already is a layer.

Reply:
Not sure what you mean by devnet. But all code i run locally uses centauri prefix

Reply:
Never used anything else

Reply:
@RustNinja we should also change it for testnet too if you haven't
 
 
 change from layer -> pica

Reply:
Yes. so the same code will be applied to test net.
Here is the: "Convert prefix from centauri to pica" text related just to command description.